### PR TITLE
Multiline class output fix

### PR DESCRIPTION
**Pull request by funny-family**
### 🤔 What is the nature of this change?

- [ ] New feature
- [x] Fix bug
- [ ] Style optimization
- [ ] Code style optimization
- [ ] Performance optimization
- [ ] Build optimization
- [ ] Refactor code or style
- [ ] Test related
- [ ] Other

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix multi line class output. Close issue: https://github.com/vuejs/babel-plugin-jsx/issues/569     |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed (not needed)
- [x] Demo is updated/provided or not needed (not needed)
- [x] TypeScript definition is updated/provided or not needed (not needed)
- [x] Changelog is provided or not needed (need)

### CODE REVIEW

**Comment by funny-family** [comment]
@Amour1688 cc

File: packages/babel-plugin-jsx/src/transform-vue-jsx.ts [hunk 1]
```diff
@@ -86,6 +86,15 @@ const buildProps = (path: NodePath<t.JSXElement>, state: State) => {
 
         const attributeValue = getJSXAttributeValue(prop, state);
 
+        if (name === 'class' && attributeValue?.type === 'StringLiteral') {
```
Comment (sxzz) [inline packages/babel-plugin-jsx/src/transform-vue-jsx.ts:89]: Not only for `class`, but for all properties.
https://babeljs.io/repl#?browsers=&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=MYewdgzgLgBAggBwTAvDAFASlQPgwKBhgB4ATASwDdCiYoBTaFAIhtpgDMQQ3aAjAIYAnXqyI5eACXoAbGSBgB1EEJmkaxAPQVKEzPiA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=react&prettier=false&targets=&version=7.18.1&externalPlugins=&assumptions=%7B%7D

**Review by sxzz — CHANGES_REQUESTED** [review]

File: packages/babel-plugin-jsx/src/transform-vue-jsx.ts (hunk 1 above)
Comment (funny-family) [reply packages/babel-plugin-jsx/src/transform-vue-jsx.ts]: @sxzz, fixed

Comment (sxzz) [reply packages/babel-plugin-jsx/src/transform-vue-jsx.ts]: You just reverted the changes you made before.

Comment (funny-family) [reply packages/babel-plugin-jsx/src/transform-vue-jsx.ts]: My bad. Fixed. Again.

File: packages/babel-plugin-jsx/src/transform-vue-jsx.ts [hunk 2]
```diff
@@ -79,12 +79,20 @@ const buildProps = (path: NodePath<t.JSXElement>, state: State) => {
   const mergeArgs: (t.CallExpression | t.ObjectExpression | t.Identifier)[] =
     [];
   const { mergeProps = true } = state.opts;
+
```
Comment (sxzz) [inline packages/babel-plugin-jsx/src/transform-vue-jsx.ts:82]: Please revert empty changes.

**Review by sxzz — CHANGES_REQUESTED** [review]
```
const App = () => (
  <div
    hello="a  b"
  >
    Hello World
  </div>
)
```
It should be `hello: "a  b"`

File: packages/babel-plugin-jsx/src/transform-vue-jsx.ts (hunk 2 above)
Comment (funny-family) [reply packages/babel-plugin-jsx/src/transform-vue-jsx.ts]: Done.

**Review by sxzz — COMMENTED** [review]
I think we can use the same logic in `transformJSXText` from `utils.ts`.

**Comment by sxzz** [comment]
Thanks for the PR. We can directly use `transformJSXText` function 8ed497de1dded45e51535bd55ac5b91e9718d5a9

**Pull request closed without merging**